pkgs/tools/security/vulnix/default.nix

```diff
@@ -19,6 +24,9 @@ python3Packages.buildPythonApplication rec {
     pytest-flake8
   ];
 
+  # indirect dependency ZConfig (from ZODB) does not support Python 3.8
+  disabled = python.passthru.pythonAtLeast "3.8";
```

Comment:
Actually, you could use:

```python
disabled = python3Packages.python.passthru.pythonAtLeast "3.8";
```

this will avoid adding a dependency and ensure that `python` and `python3Packages.python` refer to the same python

Reply:
to make it even simpler:
```
disabled = python3Packages.pythonAtLeast "3.8";
```